### DISCUSSION TITLE
스테이지 관련 기획이 더 필요한 부분

### DISCUSSION THREAD

**Discussion by seong0929** [SeoulTechTCPGame/HumanSurvival#119] [Ideas]
- 캐릭터 설명창 구현을 완료했습니다. 캐릭터 특징과 스탯을 정리 부탁드립니다.
-  "캐릭터 구매 구현(잠금 해제, 미구매 실루엣, 구매)" 이 todolist에 관해서 정보 관련 저장 시스템이 필요합니다. to do list 작성을 부탁드립니다.
-  "스테이지에 횃불 자동스폰 적용" 횃불을 포괄할 더 큰 이름이 없을까요?

**Comment by rapa1130** [comment]
## Q 캐릭터 설명창 구현을 완료했습니다. 캐릭터 특징과 스탯을 정리 부탁드립니다.

**A** 제가 캐릭터 특징 및 스탯 정리에 대한 담당자는 아니지만 지난 주 기획팀 회의에서 분담한 업무 중 하나입니다. 조만간 완성될 것으로 보입니다. 

## Q "캐릭터 구매 구현(잠금 해제, 미구매 실루엣, 구매)" 이 todolist에 관해서 정보 관련 저장 시스템이 필요합니다. to do list 작성을 부탁드립니다.
**A** 확인했습니다. 


## Q "스테이지에 횃불 자동스폰 적용" 횃불을 포괄할 더 큰 이름이 없을까요?
**A** '맵 오브젝트' 정도가 떠오릅니다만, 이는 중의적으로 사용될 위험이 있어 보입니다. 다만, 뱀파이어 서바이벌을 조금이라도 해본다면 횃불이 무엇을 의미하는지는 모를 수가 없습니다. 따라서 의미하는 바를  전달하기엔 횃불이 가장 괜찮은 대안이라 생각합니다.
